sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -168,6 +173,24 @@ public CryptographyAsyncClient buildAsyncClient() {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public KeyEncryptionKey buildKeyEncryptionKey(String keyId) {
+        this.keyId = keyId;
+        return buildClient();
```

Comment:
Oh, I see you are building a client and returning. Shouldn't the return type then be a `KeyEncryptionKeyClient`?

Reply:
not an issue, as discussed